base/inc/AdePT/BVHNavigator.h

```diff
@@ -104,7 +170,17 @@ class BVHNavigator {
 
     if (pvol->GetDaughters().size() > 0) {
       auto bvh = vecgeom::BVHManager::GetBVH(pvol->GetLogicalVolume()->id());
-      bvh->CheckDaughterIntersections(localpoint, localdir, step, pvol, hitcandidate);
```

Comment:
Can you please fix the issue of not passing the correct `last_exited` in a separate PR that we can validate on its own?

Reply:
https://github.com/apt-sim/AdePT/pull/262